app/src/test/java/org/simple/clinic/util/HaversineTest.kt

```diff
@@ -0,0 +1,19 @@
+package org.simple.clinic.util
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.simple.clinic.location.Coordinates
+
+class HaversineTest {
+
+  @Test
+  fun `haversine formula should work correctly`() {
+    val start = Coordinates(latitude = 1.908537, longitude = 73.537524)
+    val end = Coordinates(latitude = 59.299800, longitude = 18.209118)
+    val maldivesToStockholm = Haversine.distance(start, end)
+    val stockholmToMaldives = Haversine.distance(start, end)
+
+    assertThat(maldivesToStockholm).isEqualTo(Distance.ofKilometers(7939.655950402867))
```

Comment:
I don't want to handle flakiness until we start seeing them. I could use `StrictMath` so that the calculations don't change on different JVMs. What do you think?

Reply:
Not too sure of this because then we're using `StrictMath` not because we actually need it, but because we want to avoid flakiness in the tests. In addition, that will take care of one type of flakiness, but not other kinds (like changing the diameter constant to a slightly different value, etc).

I am fine with leaving it as is for now, and move to a different style of assertions if we need to in the future.

Reply:
Okay, I'll leave this for now